MIGRATING.md

```diff
@@ -59,6 +58,9 @@
     - `GooglePayConfig` now requires instantiated `PaymentConfiguration`
     - `CardUtils` is now `internal`
     - `StripeTextUtils` has been removed
+    - `Stripe.confirmPayment` and `Stripe.confirmSetupIntent` will now use Custom Tabs by default
+      when performing 3DS1 Authentication.  If Custom tabs are not available on the phone the SDK
+      will use a WebView.
```

Comment:
1. Maybe worth linking to https://developer.chrome.com/docs/android/custom-tabs/overview/ for Custom Tabs.
2. If Custom Tabs aren't available, the device browser will be used.

Reply:
Is a webview never used? @mshafrir-stripe

Reply:
yep! singleTask works for chrome tabs as well.

Reply:
WebView would be used with `handleNextActionForPayment` (i.e. backend confirmation). It should no longer be used for client-side confirmation.